src/azurelinux/3.0/net10.0/source-build-test/amd64/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM mcr.microsoft.com/dotnet/sdk:10.0-preview-azurelinux3.0-amd64
```

Comment:
We should use a multi-stage Dockerfile instead so that scancode is installed in a separate stage and then copied to the final stage. This saves about 300 MB of disk space in the final image.